ping/tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py37
+envlist =
+    {py27,py37}
+    flake8
+
+[testenv]
+dd_check_style = true
+usedevelop = true
+platform = linux|darwin|win32
+deps =
+    datadog-checks-base[deps]
+    -rrequirements-dev.txt
+passenv =
+    DOCKER*
+    COMPOSE*
+commands =
+    pip install -r requirements.in
+    pytest -v
+
+[testenv:flake8]
```

Comment:
Let's remove everything from this line, it's not needed anymore with `dd_check_style`